src/gui/torrentcontentmodel.cpp

```diff
@@ -116,27 +116,8 @@ namespace
     };
 #endif // QBT_PIXMAP_CACHE_FOR_FILE_ICONS
 
-#if defined(Q_OS_WIN)
```

Comment:
You should also remove related windows specific headers.

Reply:
> You should also remove related windows specific headers.

Done.